src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,124 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_NODES_LENGTH = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private int loaded = (int) (DEFAULT_NODES_LENGTH * LOAD_FACTOR);
```

Comment:
```suggestion
    private int threshold;
```

let's initialize non-static fields in constructor